src/utils.js

```diff
@@ -164,3 +164,23 @@ export function isThisYear(date: Date) {
   const now = new Date()
   return now.getUTCFullYear() === date.getUTCFullYear()
 }
+
+// eslint-disable-next-line flowtype/no-weak-types
+export function makeRelativeFormatter(options: any): () => ?any {
+  let format
+  return function() {
+    if (format) return format
+    if ('Intl' in window && 'RelativeTimeFormat' in window.Intl) {
+      try {
+        // eslint-disable-next-line flowtype/no-flow-fix-me-comments
+        // $FlowFixMe: missing RelativeTimeFormat type
+        format = new Intl.RelativeTimeFormat(undefined, options)
```

Comment:
I think we want to pass `en` as the locale until we add a `locale` attribute to the custom element so it's configurable by the host app. That will prevent this text from appearing in the default locale for the browser with the rest of the text appearing in English.

Reply:
Yeah, I'm down to support that, though I was just stealing from our existing `makeFormatter`. 

https://github.com/github/time-elements/blob/4e6f95a34fc897626e794ed78f0f45701043de67/src/utils.js#L102

Will that need to change as well? Should I do this in a separate PR? Starting to wonder how this change is going to ship. It might be a major version bump. So could bundle this and that locale change in one release.

Reply:
I wonder if the [runtime's default locale](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation) is hardcoded to `en` in browsers and this is different from the user's preferred language setting in the operating system. Anyway, since undefined is already working for DateTimeFormat, we can use it here too.

A major version release would be appropriate since this changes the text output slightly.

We could couple that with `<relative-time locale="…">` if it's not too much work. The only weird thing is that the locale attribute would only apply to browsers with `RelativeTimeFormat` support. The older code path would still use the hardcoded English text.